docs/structured-streaming-kafka-integration.md

```diff
@@ -416,6 +416,24 @@ The following configurations are optional:
 </tr>
 </table>
 
+### Consumer Caching
+
+It's time consuming to initialize Kafka consumers especially in streaming scenarios where processing time is key factor.
+Because of this Spark caches Kafka consumers on executors. The caching key built-up from the following information:
+* Topic name
+* Topic partition
+* Group ID
+
+The size of the cache is limited by <code>spark.kafka.consumer.cache.capacity</code> (default: 64).
+If the mentioned threshold reached it tries to remove the least-used entry which is currently not in use.
```

Comment:
If this threshold is reached, 
entry that is currently

Reply:
Fixed.